### PR TITLE
Option to show/hide boosts for a followed user

### PR DESCRIPTION
**Pull request by cosmin**
Support for hiding someone while hiding their boosts. The AP schema refers to boosts as reblogs, as does the client API, so for consistency I also called the field reblogs on the follow schema.

Currently this only implements this on the schema and in the API, no UI is exposed to show/hide boosts on a follow.

### CODE REVIEW

File: users/services/identity.py [hunk 1]
```diff
@@ -75,7 +82,7 @@ def mastodon_json_relationship(self, from_identity: Identity):
             "requested": False,
             "domain_blocking": False,
             "endorsed": False,
-            "note": "",
+            "note": follow.note or "",
```
Comment (cosmin) [inline users/services/identity.py:85]: this is unrelated to show/hide boosts but it's another property of the follow object which was previously unavailable. I can split this into a separate PR if that would be cleaner.

**Review by andrewgodwin — CHANGES_REQUESTED** [review]
I know this is annoying, but I'm going to request that it is called "boosts" not "reblogs" for internal codebase consistency

File: users/services/identity.py (hunk 1 above)
Comment (andrewgodwin) [reply users/services/identity.py:85]: This is fine!

**Review by andrewgodwin — APPROVED** [review]
Looks good, but I'm going to wait until after 0.6.1 is released to merge this as it has a migration.

**Pull request merged**